ci_framework/roles/hive/files/add_virt_net.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+# Configure the virtual network.
+#
+# Usage:
+#
+#   add_virt_net <net_name>
+#
+# where
+#     <net_name>    The name of the network
+#
+set -euo pipefail
+
+NET_NAME=$1
+
+check_exists=$(virsh net-list --all | grep ${NET_NAME} | wc -l) || true
```

Comment:
```suggestion
check_exists=$(virsh net-list --all | grep -c ${NET_NAME}) || true
```

`-c` will count the occurrences. One less pipe, that's better with grep, especially when we set `pipefail`.

Reply:
Thanks. Something new that I learnt. Incorporated it :)